docs/guide/helpers.md

```diff
@@ -69,6 +69,18 @@ class UseHelpersComponent < ViewComponent::Base
 end
 ```
 
+Methods defined in helper modules that are not available in the component can be included individually using the `from:` keyword:
+
+```ruby
+class UserComponent < ViewComponent::Base
+  use_helpers :icon, from: IconHelper
```

Comment:
I guess it would make sense to support the singular form `use_helper` as well?

Reply:
@Spone I have added this recommendation, as it seems like a great fit